src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -178,27 +63,48 @@
                   :number-of-lines 5}
       text]]))
 
+(def ^:private kind->props-fn {:bold   (constantly {:style {:font-weight :bold}})
```

Comment:
not a big deal, but any chance we can move the map to the next line? it would probably fit into 80 chars then :)

Reply:
Of course, will do :)